packages/ra-ui-materialui/src/layout/NotFound.js

```diff
@@ -49,13 +49,13 @@ const NotFound = ({
     className,
     classes: classesOverride,
     title,
-    location,
+    location, // not used, but don't want to pass to div
```

Comment:
prefer a `sanitizeRestProps` function that cleans up the props passed to the div.